packages/file/src/types/hfile.coffee.md

```diff
@@ -0,0 +1,188 @@
+
+`nikita.file.types.hfile`
+
+HFile is an XML format used accros Hadoop components which contains keys and
+value properties.
+
+## Schema
+
+    schema =
+      type: 'object'
+      properties:
+        'merge':
+          type: 'boolean'
+          description: """
+          Read the target if it exists and merge its content, optional.
+          """
+        'source':
+          type: ['object', 'string']
+          description: """
+          Default configuration properties or the path to a default
+          configuration file to get initial value from, optional.
+          """
+        'target':
+          type: 'string'
+          description: """
+          Configuration file where to write, required.
+          """
+        'properties':
+          type: 'object'
+          description: """
+          Configuration properties to write, required.
+          """
+        'transform':
+          typeof: 'function'
+          description: """
+          User defined function used to transform properties.
+          """
+        # File configuration properties
+        'backup':
+          $ref: 'module://@nikitajs/file/src/index#/properties/backup'
+        'backup_mode':
+          $ref: 'module://@nikitajs/file/src/index#/properties/backup_mode'
+        'eof':
+          $ref: 'module://@nikitajs/file/src/index#/properties/eof'
+        'encoding':
+          $ref: 'module://@nikitajs/file/src/index#/properties/encoding'
+        'uid':
+          $ref: 'module://@nikitajs/file/src/index#/properties/uid'
```

Comment:
Missing `gid` as well as `mode`. We spoke about a lot of other properties including `local`, `unlink`...